askama_shared/src/lib.rs

```diff
@@ -257,19 +258,71 @@ where
     vals.iter().map(|s| s.to_string()).collect()
 }
 
+/// Whitespace handling of the input source.
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum Strip {
+    /// Don't strip any spaces in the input.
+    None,
+    /// Remove a single single newline at the end of the input. This is the default.
+    Tail,
+    /// Remove all whitespaces at the front and back all lines, and remove empty lines.
+    Trim,
```

Comment:
Maybe call this `TrimLines` to avoid confusion as with @vallentin?

Reply:
I renamed it into TrimLines / "trim-lines".